File: 프로그래머스/lv0/120823. 직각삼각형 출력하기/직각삼각형 출력하기.swift
import Foundation

let n = readLine()!.components(separatedBy: [" "]).map { Int($0)! }

func solution(n: Int) -> String {
    var answer = ""
    for i in 0..<n {
        for j in 0..<n {
            if j <= i {
                answer += "*"
            }
        }
        answer += "\n"
    }
    return answer
}

print(solution(n: n[0]))

File: 프로그래머스/lv0/120823. 직각삼각형 출력하기/직각삼각형 출력하기.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    input = line.split(' ');
}).on('close', function () {
    const n = Number(input[0]);
    let answer = '';
    for (let i = 0; i < n; i++) {
        answer += '*'.repeat(i + 1)
        answer += '\n'
    }
    console.log(answer);
});